src/main/resources/META-INF/persistence.xml

```diff
@@ -2,7 +2,7 @@
 <persistence version="2.2" xmlns="http://java.sun.com/xml/ns/persistence" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/persistence http://xmlns.jcp.org/xml/ns/persistence/persistence_2_2.xsd">
   <persistence-unit name="hmisPU" transaction-type="JTA">
     <provider>org.eclipse.persistence.jpa.PersistenceProvider</provider>
-    <jta-data-source>jdbc/ruhunu</jta-data-source>
+    <jta-data-source>jdbc/southernlanka</jta-data-source>
```

Comment:
Is this because this version is targeted to Southern Lanka Hospital or has been committed with other changes unintentionally?
Either way, we can follow another method without changing these configuration files that may affect other developers when they pull these changes.

You can mark the comment as resolved. Let's have a discussion later on how to proceed with this. Thank you!